src/content/contribute/writing-a-scaffold.md

```diff
@@ -0,0 +1,400 @@
+---
+title: Writing a Scaffold
+sort: 3
+contributors:
+  - pranshuchittora
+---
+
+Welcome to an demonstration of the new `webpack init` feature! To view what we are building today, simply do `webpack init webpack-scaffold-demo`. This demo is to show you how to build your own webpack scaffold. Let's start by creating an file named `generator.js`.
```

Comment:
1. "This demo is to show you", may be better "This demo will show you" ?
2. "an file named" -> "a file named"